lighthouse-core/test/results/sample_v2.json

```diff
@@ -2232,7 +2232,6 @@
             "node": {
               "type": "node",
               "selector": "h2",
-              "path": "3,HTML,1,BODY,5,DIV,0,H2",
```

Comment:
something odd is going on here, these shouldn't be dropping.

Reply:
ah, need to rename `path` to `devtoolsNodePath` in the accessibility artifact in lighthouse-core/test/results/artifacts/artifacts.json